packages/react-components/react-card/src/components/CardPreview/useCardPreviewStyles.ts

```diff
@@ -14,8 +14,9 @@ const useStyles = makeStyles({
   root: {
     position: 'relative',
 
-    '> *': {
+    [`> :not(.${cardPreviewClassNames.logo})`]: {
```

Comment:
Previously wrongfully applying to any child element, which would include the floating logo slot.

Reply:
also improves paint perf, nice catch